### PR TITLE
Updated FAQ.md with the correct links

### PR DESCRIPTION
**Pull request by nikkibeesetti**
Two links regarding plugins were broken and now they have been fixed.

#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -11,15 +11,11 @@ brand.
 
 ### Is Backstage a monitoring platform?
 
-No, but it can be! Backstage is designed to be a developer portal for all your
-infrastructure tooling, services, and documentation. So, it's not a monitoring
-platform — but that doesn't mean you can't integrate a monitoring tool into
-Backstage by writing
-[a plugin](https://github.com/spotify/faq#what-is-a-plugin-in-backstage).
+No, but it can be! Backstage is designed to be a developer portal for all your infrastructure tooling, services, and documentation. So, it's not a monitoring platform — but that doesn't mean you can't integrate a monitoring tool into Backstage by writing [a plugin](https://github.com/spotify/faq#what-is-a-plugin-in-backstage).
```
Comment (stefanalund) [inline docs/FAQ.md:14]: This should be /spotify/backstage

File: docs/FAQ.md [hunk 2]
```diff
@@ -99,15 +95,7 @@ plugins. ​
...
-to integrate different systems into Backstage's frontend, so that the developer
-gets a consistent UX, no matter what tool or service is being accessed on the
-other side. ​ Each plugin is treated as a self-contained web app and can include
-almost any type of content. Plugins all use a common set of platform APIs and
-reusable UI components. Plugins can fetch data either from the backend or an API
-exposed through the proxy. ​ Learn more about
-[the different components](https://github.com/spotify/backstage#overview) that
-make up Backstage. ​
+By far, our most-used plugin is our TechDocs plugin, which we use for creating technical documentation. Our philosophy at Spotify is to treat "docs like code", where you write documentation using the same workflow as you write your code. This makes it easier to create, find, and update documentation. We hope to release [the open source version](https://github.com/spotify/backstage/issues/687) in the future. (See also: "[Will Spotify's internal plugins be open sourced, too?](https://github.com/spotify/faq#will-spotifys-internal-plugins-be-open-sourced-too)" above)
```
Comment (stefanalund) [inline docs/FAQ.md:98]: Same issue as above.

**Review by stefanalund — CHANGES_REQUESTED** [review]

File: docs/FAQ.md (hunk 1 above)
Comment (AlleyWei) [reply docs/FAQ.md:14]: @nikkibeesetti @stefanalund this should be the link: 

https://github.com/spotify/backstage/blob/master/docs/FAQ.md#what-is-a-plugin-in-backstage

File: docs/FAQ.md (hunk 2 above)
Comment (AlleyWei) [reply docs/FAQ.md:98]: @nikkibeesetti @stefanalund This is the correct link for this:

https://github.com/spotify/backstage/blob/master/docs/FAQ.md#what-is-a-plugin-in-backstage

**Review by nikkibeesetti — COMMENTED** [review]
Updated broken links

File: docs/FAQ.md (hunk 2 above)
Comment (AlleyWei) [reply docs/FAQ.md]: @nikkibeesetti sorry that was my bad, this link should be: 

https://github.com/spotify/backstage/blob/master/docs/FAQ.md#will-spotifys-internal-plugins-be-open-sourced-too

Comment (Rugvip) [reply docs/FAQ.md]: You can use just `/docs/FAQ.md#will-spotifys-internal-plugins-be-open-sourced-too` as well

**Review by stefanalund — APPROVED** [review]

**Pull request merged**